surveys/sphinx-survey-2025.yml

```
@@ -119,25 +108,43 @@ questions:
   - "Testing engineer"
 
 # Motivation: Understand the constellation of tools for documentation this responder works with.
-- title: "What documentation engines do you use?"
+- title: "What documentation tools/engines do you use?"
   multiple: true
   other: true
   choices:
   - "Astro"
+  - "Confluence"
   - "Docusaurus"
   - "Eleventy"
   - "Hugo"
   - "JupyterBook"
+  - "JSDoc"
   - "Mkdocs"
```

```suggestion
  - "MkDocs"
```

matching the style they use on their site